jobs/python.setup.py.yml

```diff
@@ -18,76 +19,76 @@ parameters:
   pythonUseVersion: "3.6"
 
 jobs:
-- job: 'Linux'
-  strategy:
-    matrix:
-      Python35:
-        python.version: '3.5'
-      Python36:
-        python.version: '3.6'
-      Python37:
-        python.version: '3.7'
-  pool:
-    vmImage: 'ubuntu-16.04'
-  steps:
-  - template: ../steps/python/python.build.yml
+  - job: 'Linux'
```

Comment:
Why these formatting changes?

Reply:
It was not actually correctly formatted yaml

Reply:
It was not wrong, just different but I agree, let's keep it consistent and good to have a tool doing it for us 👍